myuw/dao/instructor_schedule.py

```diff
@@ -71,6 +72,30 @@ def get_current_quarter_instructor_schedule(request):
     return schedule
 
 
+def get_future_quarter_instructor_schedule(request):
```

Comment:
Our approach with non-current quarters in the DAO is to pass in a term:

https://github.com/uw-it-aca/myuw/blob/master/myuw/dao/schedule.py#L40